content/developer/howtos/javascript_views.rst

```diff
@@ -0,0 +1,260 @@
+=====
+Views
+=====
+
+Customizing an existing view
```

Comment:
split this into 2 how tos: "Subclass an existing javascript view" and "Create a new view from scratch"